itou/templates/signup/prescriber_check_already_exists.html

```diff
@@ -0,0 +1,158 @@
+{% extends "layout/content_small.html" %}
+{% load bootstrap4 %}
+{% load static %}
+{% load format_filters %}
+
+{% block title %}Prescripteur/Orienteur - Inscription{{ block.super }}{% endblock %}
+
+{% block content %}
+
+    <h1>
+        Inscription
+        <small class="text-muted">Prescripteur/Orienteur</small>
+    </h1>
+
+    <h2>
+        Votre organisation est-elle déjà inscrite ?
+    </h2>
+
+    <p class="mb-6">
+        <a href="{% url 'signup:prescriber_pole_emploi_safir_code' %}">Je travaille pour Pôle emploi</a>
+    </p>
+    <form method="post" action="" class="js-prevent-multiple-submit">
+
+        {% csrf_token %}
+
+        {% bootstrap_form form alert_error_type="all" %}
+
+        {% buttons %}
+            <button type="submit" class="btn btn-primary">Rechercher</button>
+        {% endbuttons %}
+
+    </form>
+
+    {# Display link only before the search otherwise an organization is put in session and no longer allows creation without an organization  #}
+    {% if not prescriber_orgs_with_members_same_siret and not prescriber_orgs_with_members_same_siren %}
+    <p class="mt-3">
+        <a href="{% url 'signup:prescriber_user' %}">Je ne fais partie d'aucune organisation</a>
+    </p>
+    {% endif %}
+
+    {% if prescriber_orgs_with_members_same_siret %}
+        <div class="mt-5">
+            <h3>Organisation(s) déjà inscrite(s) avec ce SIRET</h3>
+            <p>Par mesure sécurité, vous devez obtenir une invitation pour rejoindre une organisation déjà inscrite.</p>
```

Comment:
```suggestion
            <p>Par mesure de sécurité, vous devez obtenir une invitation pour rejoindre une organisation déjà inscrite.</p>
```

Il manque un mot ici.

Reply:
Merci ! Tu te demandes des fois comment tu peux ne pas l'avoir vu en le tapant et en relisant..